packages/ui/fe-bundle/src/pages/[host]/[[...slug]].tsx

```diff
@@ -8,34 +8,58 @@ import Head from "next/head";
 import { REGISTRY_SERVICE } from "../../service";
 import { getSlugFromUrl } from "../../url-path-resolver/getSlugFromUrl";
 import { UrlPathResolver } from "../../url-path-resolver/UrlPathResolver";
+import { generateFontFaces, loadDocTypography } from "../../utils/theme/loadDocsTypography";
 
 const inter = Inter({ subsets: ["latin"] });
 
 export declare namespace Docs {
     export interface Props {
         docs: FernRegistryDocsReadV2.LoadDocsForUrlResponse;
         resolvedUrlPath: ResolvedUrlPath;
+        typographyStyleSheet?: string;
         nextPath: ResolvedUrlPath | null;
         previousPath: ResolvedUrlPath | null;
     }
 }
 
-export default function Docs({ docs, resolvedUrlPath, nextPath, previousPath }: Docs.Props): JSX.Element {
+function Typography({ stylesheet }: { stylesheet: string }) {
     return (
-        <main className={inter.className}>
-            <Head>
-                {docs.definition.config.title != null && <title>{docs.definition.config.title}</title>}
-                {docs.definition.config.favicon != null && (
-                    <link rel="icon" id="favicon" href={docs.definition.files[docs.definition.config.favicon]}></link>
-                )}
-            </Head>
-            <App
-                docs={docs}
-                resolvedUrlPath={resolvedUrlPath}
-                nextPath={nextPath ?? undefined}
-                previousPath={previousPath ?? undefined}
-            />
-        </main>
+        // eslint-disable-next-line react/no-unknown-property
+        <style id="fern-font-stylesheet" jsx global>
```

Comment:
what do `jsx` and `global` do?

Reply:
When using `<styled jsx global>`, the `global` attribute injects the stylesheet contents inside the document `head`. The `jsx` attribute is a compile-time directive that informs Next.js to process this stylesheet as a [`styled-jsx`](https://nextjs.org/docs/pages/building-your-application/styling/css-in-js) primitive